recipes-security/optee/optee-fiovb_git.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "OP-TEE Foundries.IO Verified Boot"
+HOMEPAGE = "https://github.com/ldts/optee-fiovb"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=92d506fc36dda404ceb608cdc34b7a99"
+
+DEPENDS = "optee-client virtual/optee-os python-pycrypto-native"
+
+inherit pythonnative
+
+SRC_URI = "git://github.com/ldts/optee-fiovb.git "
```

Comment:
Push the main repo to the foundriesio github project, and also feel free to remove all the android related makefiles from it, as we don't use them.

Reply:
funny I just did the cmake and android and saw this request (so sure, makes sense) ok will clean current changes as a single commit and then fork to foundries